### ISSUE TITLE
Got error: Your Functions script is over the 1 MiB size limit (workers.api.error.script_too_large)

### ISSUE THREAD

**Issue by s1lviu** [cloudflare/next-on-pages#68]
Hi, I try to deploy a (tiny) Next app to Cloudflare pages, all seems ok, but in the end, I receive the following:

```
15:18:09.647 | ⚡️ Generated '.vercel/output/static/_worker.js'.
-- | --
15:18:09.749 | Finished
15:18:09.749 | Found _worker.js in output directory. Uploading.
15:18:09.791 | Validating asset output directory
15:18:19.787 | Success: Assets published!
15:18:23.068 | Error: Failed to publish your Function. Got error: Your Functions script is over the 1 MiB size limit (workers.api.error.script_too_large)
```
PS: The build command has the minify flag specified: `npx @cloudflare/next-on-pages --experimental-minify`
Thanks!

**Comment by sgoodluck** [comment]
Bump (and watching). I'm about to start down the path of deploying a next app on cloudflare pages as well. Will let you know if I run into similar issues or avoid it. Hopefully someone who's done this chimes in too!

The docs state that [a worker has to be under 1MB](https://developers.cloudflare.com/workers/platform/limits/#worker-size). Are y

**Comment by erickeira** [comment]
Comigo sempre me foi mostrado essa mensagem,  hoje eu usei o comandado npx @cloudflare/next-on-pages --experimental-minify localmente e subi a pasta .vercel, assim o Cloudflare conseguiu buildar sem criar as functions

**Comment by GregBrimble** [comment]
Closed as dupe of #47
